src/ui/FieldTable/FieldTable.ts

```diff
@@ -246,11 +244,12 @@ export class FieldTable extends Component {
     if (!anim) {
       $$(this.toggleContainer).addClass('coveo-no-transition');
     }
+
     if (visible) {
       this.toggleContainer.style.display = 'block';
-      this.toggleContainer.style.height = this.toggleContainerHeight + 'px';
+      this.toggleContainer.style.height = this.containerHeight;
     } else {
-      this.toggleContainer.style.height = this.toggleContainerHeight + 'px';
+      this.toggleContainer.style.height = this.containerHeight;
       this.toggleContainer.style.height = '0';
```

Comment:
Since the height is set twice here, won't this second line override the first one?

Reply:
Indeed, I removed the first `height` assignment. Thanks,